front/src/modules/pipeline/components/BoardActionBarButtonDeletePipelineProgress.tsx

```diff
@@ -1,39 +1,59 @@
 import { getOperationName } from '@apollo/client/utilities';
```

Comment:
I think you can move all these files into @/ui/board/ (like for table). It will make clear that we don't want to talk about pipelines, companies or or pipelinestages here!